src/java.desktop/unix/classes/sun/awt/UNIXToolkit.java

```diff
@@ -521,6 +520,18 @@ public boolean isRunningOnWayland() {
     // application icons).
     private static final WindowFocusListener waylandWindowFocusListener;
 
+    private static boolean containsWaylandWindowFocusListener(Window window) {
+        if (window == null) {
+            return false;
+        }
+        for (WindowFocusListener focusListener : window.getWindowFocusListeners()) {
```

Comment:
```suggestion
        }

        for (WindowFocusListener focusListener : window.getWindowFocusListeners()) {
```
I suggest adding a blank line to visually separate the early return logic from the common case one.